modules/fts/pages/fts-search-response-facets.adoc

```diff
@@ -11,57 +11,83 @@ Facet-query results may not equal the total number of documents across all bucke
+For example, size - 3 => facets results returns the top 3 categories across all partitions and merges them as the final result.
++
+Varying size value varies the count value of each facet and the “others” value as well. This is due to the fact that when the size is varied, some of the categories fall out of the top “n” and into the “others” category.
++
+It is recommended to keep the size reasonably large, close to the number of unique terms to get consistent results.
+
+* *Numeric Range Facet*: A numeric range facet works by the user defining their own buckets (numeric ranges).
++
 The facet then counts how many of the matching documents fall into a particular bucket for a particular field.
++
+Along with the two fields from term facet, “numeric_ranges” field has to include all the numeric ranges for the faceted field.
++
+“Numeric_ranges” could possibly be an array of ranges and each entry of it must specify either min, max or both for the range.
+
+** *Name*: Name for the facet.
+
+** *Min*: The lower bound value of this range.
 
-* Along with the two fields from term facet, “numeric_ranges” field has to include all the numeric ranges for the faceted field.
-* “Numeric_ranges” could possibly be an array of ranges and each entry of it must specify either min, max or both for the range.
-** Name - name for the facet	
-** Min - the lower bound value of this range.
-** Max - the upper bound value of this range.
+** *Max*: The upper bound value of this range.
 
-* *Date Range Facet* - same as numeric, but on dates instead of numbers.
+* *Date Range Facet*: The Date Range facet is same as numeric facet, but on dates instead of numbers.
 Full text search and Bleve expect dates to be in the format specified by https://www.ietf.org/rfc/rfc3339.txt[RFC-3339^], which is a specific profile of ISO-8601 that is more restrictive.
++
+Along with the two fields from term facet, “date_ranges” field has to include all the numeric ranges for the faceted field.
++
+The facet ranges go under a field named “date_ranges”. 
++
+“date_ranges” could possibly be an array of ranges and each entry of it must specify either start, end or both for the range.
+
+** *Name*: Name for the facet.
+
+** *Start*: Start date for this range.
 
-* Along with the two fields from term facet, “date_ranges” field has to include all the numeric ranges for the faceted field.
-* The facet ranges go under a field named “date_ranges”. 
-* “date_ranges” could possibly be an array of ranges and each entry of it must specify either start, end or both for the range.
-** Name - name for the facet
-** Start - start date for this range
-** End - end date for this range
+** *End*: End date for this range.
 
 NOTE: Most of the time, when building a term facet, you must use the keyword analyzer. Otherwise, multi-term values are tokenized, which might cause unexpected results.
 
 == Example
 
-*Term Facet* - computes facet on the type field which has 2 values: `beer` and `brewery`.
-+
+=== *Term Facet*
```

Comment:
Don't put bold in titles. They come up pretty bold anyway

Reply:
Updated